packages/web-react/src/components/ButtonLink/ButtonLink.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import classNames from 'classnames';
+import { WithChildren } from '../../types/main';
+import { compose } from '../../utils/compose';
+import { applyColor, applyClassNamePrefix } from '../../utils/classname';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+
+type Color = 'primary' | 'secondary' | 'tertiary';
+
+export interface ButtonLinkProps extends WithChildren {
+  /**
```

Comment:
Still thinking about these comments in props. I think that the comments are unnecessary because variable names are sufficiently self-descriptive.

I suggest to removing them.

Reply:
incorporated